modules/ephemeris/src/main/scala/gem/horizons/HorizonsClient.scala

```diff
@@ -32,18 +32,19 @@ import scala.concurrent.duration._
 object HorizonsClient {
 
   /** A global mutex for limiting concurrent connections to Horizons. */
-  private val mutex: Semaphore[IO] =
-    Semaphore[IO](1L)(implicitly, scala.concurrent.ExecutionContext.global).unsafeRunSync // note
+//  private val mutex: Semaphore[IO] =
+//    Semaphore[IO](1L)(implicitly, scala.concurrent.ExecutionContext.global).unsafeRunSync // note
```

Comment:
It turns out that this causes problems when you try to fold over a stream of multiple queries as we do when downloading a full semester of data.  We'll need another solution here.